### DISCUSSION TITLE
Ability to view files side-by-side

### DISCUSSION THREAD

**Discussion by egor-khanko** [community/community#10395] [Other Feature Feedback, Questions, & Ideas]
Hello everyone! Loving the Codesearch so far, thanks everyone for your work!
Want to ask, if there is any ability or plans to implement side-by-side file viewing? I would love to be able to just click the file link, holding ctrl, for example, and open it on the side. Even simple code viewing window (without references feature, for example) will help a lot!
Thanks in advance!

**Comment by colinwm** [comment]
Hey @egor-khanko, thanks for the suggestion, I haven't heard this before. What would you use side-by-side viewing for?

**Comment by egor-khanko** [comment]
Hi! For example, when I want to see definition of the method, and how it used at the same time. Another case I can think of is specs with described classes.
Maybe it is just a beginner thing, that I can't memorize everything right away, and It's convenient, when everything is in sight)
